ebmlite/core.py

```diff
@@ -708,19 +697,19 @@ def gc(self, recurse=False):
     def encodePayload(cls, data, length=None):
         """ Type-specific payload encoder for 'master' elements.
         """
-        result = u''
+        result = bytearray()
         if data is None:
             return result
         elif isinstance(data, dict):
-            data = data.items()
+            data = list(data.items())
```

Comment:
Casting to list does avoid the risk of the dict changing during encoding, but is it worth the memory usage? These could be *massive* elements. `data.items()` should work as-is in the loop.